src/pages/guides/theming/index.md

```diff
@@ -0,0 +1,397 @@
+- **presets** - Adds additional theme configurations on top of Tailwind's base styles. The  `venia` theme is added by default. You have the option to keep the `venia` theme and override or extend it with your own theme, or replace it completely. See [Presets](https://v2.tailwindcss.com/docs/presets) from the Tailwind documentation.
+- **purge** - The `purge` property is used to configure the [PurgeCSS](https://purgecss.com/) tool to remove unused CSS from the `index.css` file. This optimizes the production builds by reducing the size of the CSS bundle. See [Writing purgeable HTML](https://v2.tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html) and [Removing unused CSS](https://v2.tailwindcss.com/docs/optimizing-for-production#removing-unused-css) from the Tailwind documentation.
+- **plugins** - The `plugins` property configures Tailwind CSS plugins. See [Plugins](https://v2.tailwindcss.com/docs/plugins) from the Tailwind documentation.
+- **separator** - The `separator` property configures the character used to separate the variant prefixes from the utility names. Tailwind uses the colon `:` by default, but PWA Studio uses the underscore (`_`) because CSS Modules doesn't support the colon in class names. See [Separator](https://v2.tailwindcss.com/docs/configuration#separator) from the Tailwind documentation.
+- **theme** - The `theme` property is for your custom values used to override or extend the Tailwind CSS framework. See [Theme Configuration](https://v2.tailwindcss.com/docs/theme) from the Tailwind documentation.
+- **matcher** - The `matcher` variable is used to match CSS Modules classnames that use the `composes` function. This is used by PurgeCSS to remove unused CSS from the `index.css` file. See [Writing purgeable HTML](https://v2.tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html) from the Tailwind documentation.
+
+<InlineAlert slots="text" />
+
+When looking at code samples in the Tailwind docs, keep in mind that the base project configures Tailwind to use `_` as the [seperator](https://v2.tailwindcss.com/docs/configuration#separator) instead of the default colon (`:`).
+
+### Add postcss.config.js
+
+The `postcss.config.js` file is a standard configuration file used by the `postcss-loader` package. So you need to add it to your project's root and configure it to add `tailwindcss` and the `autoprefixer` as a PostCSS plugins for your app. Your `postcss.config.js` file should look like this:
+
+The `create-pwa` CLI adds and configures this file as follows:
+
+```js
+// postcss.config.js
+
+module.exports = {
+  plugins: [
+    require('autoprefixer'),
+    require('tailwindcss')('./tailwind.config.js'),
+  ],
+};
+```
+
+## Step 3: Test your configuration
+
+To test your configuration with the `[venia]` theme set as a `preset` in your config file, add the following `theme` style override and rebuild your app.
```

Comment:
At the beginning of this topic, you described that it was meant for those who didn't use the default scaffolding. I was willing to give the benefit of the doubt that step 2 was just for setting up in an alternate environment, but really, it does contain advice for changing the defaults. As of this step though, it's clear that it applies to everyone. 

Rewrite the intro and make sure it's clear that 

* Step 1 provides details about the two installation paths
* Alternate scaffolding users must go through step 2, but the happy path installers might need to make changes from the default configuration
* Steps 3 and 4 are for everyone.

Reply:
Below, in option 2, you say modifying the `config.js` file isn't the right thing to do in most cases. I recommend saying that this step is a quick test, and you might want to back out these changes when creating your own presets.